packages/build-info/tests/test-setup.ts

```diff
@@ -13,8 +13,13 @@ vi.mock('fs', async () => {
   return { default: united, ...united }
 })
 
-// cleanup after each test
-afterEach(() => {
+// cleanup after each test as a fallback if someone forgot to call it
+afterEach(async ({ cleanup }) => {
+  if (typeof cleanup === 'function') {
+    // if cleanup exists run it
```

Comment:
nit: Feels redundant?

Reply:
what do you mean with redundant? This is a safety fallback as If someone forgets to call the cleanup function on a test ;) 

if the temp dir is already cleaned up it's a noop

Reply:
Addressed in https://github.com/netlify/build/pull/4977/commits/ac28711e57721593b35986c1b39480354cba73af